canonical-web-and-design/ubuntu.com
# Share CUBE badge and microcerts to social

## Done

- Add Twitter, LinkedIn and copy to clip board share options for each microcert
- Fix certified badge share to Twitter and LinkedIn post
- Changed logic of `passed_courses = 15` to `certified_badge` incase the number of passed courses changes in the future
- Add logic to fix `"You have completed 0 microcerts..."` sentence.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube/microcerts
- Check the LinkedIn, Twitter and copy to clipbpard works
- Check different views of page to ensure the correct content is visible 


## Issue / Card

Fixes #9026 

## Screenshots
